src/test/testTransforms.ts

```diff
@@ -0,0 +1,129 @@
+/* tslint:disable:no-unused-expression */
+
+import { expect } from 'chai';
+import {
+  visit,
+  GraphQLSchema,
+  NamedTypeNode,
+  Kind,
+  GraphQLNamedType,
+  graphql,
+} from 'graphql';
+import { Request } from '../Interfaces';
+import { Transform } from '../transforms/transforms';
+import { visitSchema, VisitSchemaKind } from '../transforms/visitSchema';
+import { propertySchema } from './testingSchemas';
+import makeSimpleTransformSchema from '../transforms/makeSimpleTransformSchema';
+
+function RenameTypes(renameMap: { [originalName: string]: string }): Transform {
```

Comment:
We could make this more general and also work for namespacing if this took a function instead of an object:

```js
(type) => ({
  oldName: 'newName'
}[type])

(type) => ('NameSpace' + type)
```